client/src/Signup/Confirmation.js

```diff
@@ -1,9 +1,59 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { Divider, Typography } from 'antd'
+import LabelImportantIcon from '@material-ui/icons/LabelImportant'
 
-export default function Confirmation() {
+const { Title, Text } = Typography
+
+const userEmail = 'chelsea@pieforproviders.com'
+const pieEmail = 'noreply@pieforproviders.com'
+
+function ListItem({ children }) {
   return (
-    <div className="confirmation">
-      <h1>Check your email for confirmation</h1>
+    <div className="flex align-center justify-left mb-1">
+      <LabelImportantIcon
+        className="mr-1"
+        style={{ color: '#000', width: '16px' }}
+      />
+      {children}
     </div>
   )
 }
+
+ListItem.propTypes = {
+  children: PropTypes.element.isRequired
+}
+
+export default function Confirmation() {
+  return (
+    <>
+      <Title>Thanks for signing up!</Title>
+      <Title level={3}>We’ve sent you an email to verify your account.</Title>
+      <Divider />
+      <div className="text-left">
+        <div className="mb-2">
+          <Text>{"Didn't receive the email?"}</Text>
+        </div>
+        <ListItem>
+          <Text>
+            {`Is ${userEmail} your correct email without typos? If not, you can restart the signup process.`}
```

Comment:
JSX interpolation doesn't require the {} around the string, so it could just be 

```js
 <Text>
   Is {userEmail} your correct email without typos? If not, you can restart the signup process.
</Text>
```